docs/operational_guide/kernel_configuration.md

```diff
@@ -1,10 +1,26 @@
-Kernel Configuration
+Docker & Kernel Configuration
 ====================
 
 This document lists the Kernel tweaks M3DB needs to run well. If you are running on Kubernetes, you may use our
 `sysctl-setter` [DaemonSet](https://github.com/m3db/m3/blob/master/kube/sysctl-daemonset.yaml) that will set these
 values for you. Please read the comment in that manifest to understand the implications of applying it.
 
+## Running with Docker
+
+If running M3DB under Docker it is recommended to give M3DB the `SYS_RESOURCE` capability so that it may raise its file
```

Comment:
> If running M3DB under Docker it is recommended to give M3DB the `SYS_RESOURCE` capability so that it may raise its file limits. If running directly with Docker this can be accomplished using:

slight tweak:

> When running M3DB inside Docker, it is recommended to add the `SYS_RESOURCE` capability to the container (using the `--cap-add` argument to `docker run`) so that it can raise its file limits: